SETI/rms-opus
# 872 ui change for surface geometry searching

- Fixes #872, Related to this pull: https://github.com/SETI/pds-opus/pull/956
- Were any Django, import pipeline, table_schema, or dictionary files modified? Y
  - Database used: opus3_test_200107 (obs_surface_geometry_target.json is modified)
  - All Django tests pass: Y (some tests are modified and added)
- Were any JavaScript or CSS files modified? Y
  - JSHINT run on all affected files: Y
  - Tested on Chrome, Firefox, Safari, and iOS: Y
    (Remember to test all browser sizes)
  - Tested for race conditions (with delayed API calls if applicable): Y

Description of changes:
- Fixed the issue of blank rows in selected metadata list when switching between "Pan" and "Pandora" in surfacegeo target name.
- Fixed the issue of all radio buttons being de-selected when surfacegeo target widget is dragged.
- Remove "Surface Geometry Target Name" from the "Current Search Terms" part of select metadata.
- Make sure new input can be added/removed after selecting a new surfacegeo target.

Known problems: (Fixed)
- From https://github.com/SETI/pds-opus/pull/956
    - If you click on radio buttons rapidly, you can get the selected metadata list to screw up. This is especially true with "Pan" and "Pandora".
    - The "Surface Geometry Target Name" field should never show up in the "Current Search Terms" part of select metadata.
    - It's possible to-deselect all radio buttons by clicking just to the left of the selected button. 
- If the user switches to a new surfacegeo target, the add input button will not work in related SURFACEGEO widgets after DOM attributes are updated in place.
